core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -393,10 +402,10 @@ public static long getBytes(String key) {
   }
 
   /**
-   * @return a copy of the internal {@link Properties} of as an immutable map
+   * @return a view of the internal {@link Properties} of as an immutable map
    */
-  public static ImmutableMap<String, String> toMap() {
-    return Maps.fromProperties(PROPERTIES);
+  public static Map<String, String> toMap() {
```

Comment:
@apc999 Because we change it to an `unmodifiableMap`. Previously it copies all the entries to a new immutableMap from `Properties`, which is very slow. Now it returns a view.